### DISCUSSION TITLE
Is there an exmple of how to use the streamer at the same time as the classification demo? Has it been done?

### DISCUSSION THREAD

**Discussion by moaarv** [bitcraze/discussions#31] [Q&A]

**Comment by moaarv** [comment]
We are trying to make the streamer work at the same time as we use the classification demo on our crazyflie 2.1. Is there any way of doing this?

**Comment by knmcguire** [comment]
The facedetection example uses the streamer: https://github.com/bitcraze/aideck-gap8-examples/tree/master/examples/image_processing/FaceDetection 

Do mind that we are right in the process of converting the examples to the new CPX protocol... so maybe it would be worth to wait a few weeks until this has been fixed.

**Comment by gemenerik** [comment]
Can't wait to use it! I see that the CPX protocol is based on FreeRTOS, are you considering PUPLOS support? Or I suppose that discussion is part of converting the examples?

**Comment by moaarv** [comment]
Do you know how often the classification demo classifies per second?

**Comment by knmcguire** [comment]
I think it is about 1 or 2 hz for the classification example. It is based on mobile net v2 which is the biggest network pretty much the AIdeck can handle

@gemenerik good question! No plans yet I think, but perhaps better to start a separate thread about this for now as it is not really related to the original post :) It would be good to continue the discussion!

**Reply by moaarv** [reply]
Thank you, What do you mean by it is the biggest Network the ai deck can handle? Is it because of memory or processor power?

**Reply by knmcguire** [reply]
memory mostly. Or in other words, the biggest common network that is out there. But there are a lot of variables, like the size of the image and such.

Greenwaves has this repo where it has a list of networks that can run on the gap8: https://github.com/GreenWaves-Technologies/image_classification_networks. They are with fixed weights though.

**Comment by gemenerik** [comment]
I was curious because I think the CPX and the streamer being based on FreeRTOS and the classification demo being based on PULPOS is the biggest problem you will run into when trying to merge them, otherwise it should be quite trivial.

It's probably good to make a separate issue of it yes!
